modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -220,6 +221,41 @@ else if (classNamesFormated != null) {
 		return sb.toString();
 	}
 
+	private static Pattern _getMethodSignaturePattern(JSONObject jsonObject) {
+		String from = jsonObject.getString("from");
+
+		String regex = StringUtil.replace(from, CharPool.SPACE, "\\s+");
```

Comment:
Hi @nicolas-sss , can we use an array of old and new string for doing the replace with only one call?